pkg/image/docker_client_test.go

```diff
@@ -34,8 +34,8 @@ type FakeDockerClient struct {
 	deleteFails bool
 }
 
-func (l FakeDockerClient) Run(image string, args ...string) ([]string, error) {
-	return l.Run(image, args...)
+func (l FakeDockerClient) Run(entrypoint string, image string, args ...string) ([]string, error) {
+	return l.Run(entrypoint, image, args...)
```

Comment:
Is this a recursive call into l.Run() ?

Reply:
🤔 I think it is? I was doing global find and replace but now maybe this code path isn't being traversed at all that it doesn't error out at all.